### PR TITLE
LPS-86124 Update lockfile

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

File: modules/apps/journal/journal-web/.npmbridgerc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (jbalsas) [inline modules/apps/journal/journal-web/.npmbridgerc]: Why is this one needed? Is anyone using modules from `journal-web`? As far as I can tell, these are very internal, so it's unlikely anyone else could be using them...

Comment (diegonvs) [reply modules/apps/journal/journal-web/.npmbridgerc]: Yes, nobody is using any module from journal-web. I'll remove this.

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2047. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/a3fba93856b7~2...a3fba93856b7) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**